src/stability_sdk/client.py

```diff
@@ -475,16 +476,25 @@ def transform_and_generate(
         requests = [
             generation.Request(
                 engine_id=self._transform.engine_id,
+                requested_type=generation.ARTIFACT_TENSOR,
                 prompt=[image_to_prompt(image)],
                 transform=param,
                 extras=extras_struct,
             ) for param in params
         ]
 
-        if not self._debug_no_chains:
+        if self._debug_no_chains:
```

Comment:
I support this rearrangement. This is the code we may actually regularly look at when debugging 😄